NEWS.md

```diff
@@ -1,80 +1,116 @@
 # ggplot2 3.1.0.9000
 
-* `geom_rug()` gains a `length` option to allow for changing the length of the rug lines. (@daniel-wells, #3109)
+This is a minor release with a main emphasis on internal changes to make ggplot2 
+faster and more consistent. The few breaking changes there are will only affect
+the aesthetics of the plot (but not break code) or, potentially, extension 
+developers if they have relied on internals that have been changed.
 
-* `coord_sf()` graticule lines are now drawn in the same thickness as 
-  panel grid lines in `coord_cartesian()`, and seting panel grid
-  lines to `element_blank()` now also works in `coord_sf()` 
+## Breaking Changes
+
+* Default labels are now generated more consistently; e.g., symbols no longer
+  get backticks, and long expressions are abbreviated with `...`
+  (@yutannihilation, #2981).
+
+* `GeomRibbon` and `GeomArea` now sort the data along the x-axis in the 
+  `setup_data()` method rather than as part of `draw_group()` (@thomasp85, 
+  #3023)
+
+* `x0` and `y0` is now recognized positional aesthetics so they will get scaled 
+  if used in extension geoms and stats (@thomasp85, #3168)
+
+* Re-generate economics data. This leads to some changes in the values of all
+  columns (especially in `psavert`), but more importantly, strips the grouping
+  attributes from `economics_long`.
+
+* `coord_sf()`, `coord_map()`, and `coord_polar()` now squash `-Inf` and `Inf`
+  into the min and max of the plot (@yutannihilation, #2972).
+
+## New Features
+
+* Add Hiroaki Yutani (@yutannihilation) to the core developer group.
+
+* This release includes a range of internal changes that speeds up plot 
+  generation. None of the changes are user facing and will not break any code,
+  but in general ggplot2 should feel much faster. The changes includes, but are
+  not limited to:
+  
+  - Caching ascent and descent dimensions of text to avoid recalculating it for
+    every title.
+  
+  - Using a faster data.frame constructor as well as faster indexing into 
+    data.frames
+    
+  - Remove all dependencies on plyr and implement faster replacement functions.
+
+* `geom_polygon()` can now draw polygons with holes using the new `subgroup` 
+  aesthetic. This functionality requires R 3.6 (@thomasp85, #3128)
+
+* Aesthetic mappings now accept functions that return `NULL` (@yutannihilation,
+  #2997).
+
+* `stat_function()` now accepts rlang/purrr style anonymous functions for the 
```

Comment:
Should be alphabetised by function name

Reply:
I don't understand that?

Reply:
The bullets should be alphabetised by the name of the function they describe.

Reply:
ah